src/deps.ts

```diff
@@ -1,4 +1,10 @@
 export { readAll } from "https://deno.land/std@0.99.0/io/util.ts";
 export { BaseSlackAPIClient } from "https://deno.land/x/deno_slack_api@0.0.2/base-client.ts";
+// TODO: can keep this dependency on deno_slack_builder until the next release, then switch this over to pull this from deno-slack-hooks.
+//       otherwise we'll have to be careful on the release timing of this repo and the hooks repo (have to both appear at the same time)
+//       luckily, there are no concerns about different protocols impacting the use of createManifest imported here, as the below is only
+//       used during local-run, which in the default/legacy protocol, logging to stdout is unlimited for that specific hook.
```

Comment:
nit: Feels like the content of this TODO is less applicable after we merge

```suggestion
// TODO: Update to import from `deno-slack-hooks` instead
```